packages/components/src/DateTimePickers/DatePicker/CalendarHeader/CalendarHeader.scss

```diff
@@ -0,0 +1,11 @@
+
+.container {
+    display: flex;
+    direction: column;
+
+    .day-of-week {
+        padding: 10px;
+        width: 2.5em;
```

Comment:
This is not responsive-proof. Moreover you'll need to sync the spaces with the calendar.
You use flex, and like the calendar, you'll have exactly 7 days. Let's use flex to take the same space for each element.

```
.container {
    display: flex;
    justify-content: space-around;

    .day-of-week {
        padding: $padding-small;
        text-align: center;
    }
}
```

And we need to apply the same thing to the calendar rows

Reply:
&#10003;